op-batcher/flags/flags.go

```diff
@@ -135,3 +147,41 @@ func CheckRequired(ctx *cli.Context) error {
 	}
 	return nil
 }
+
+// CompressorKind identifies a compressor implementation.
+type CompressorKind string
+
+const (
+	// The RatioCompressorKind kind selects the batcher.RatioCompressor (see
+	// batcher.NewRatioCompressor for a description).
+	RatioCompressorKind CompressorKind = "ratio"
+	// The ShadowCompressorKind kind selects the batcher.ShadowCompressor (see
+	// batcher.NewShadowCompressor for a description).
+	ShadowCompressorKind CompressorKind = "shadow"
+)
+
+var CompressorKinds = []CompressorKind{
+	RatioCompressorKind,
+	ShadowCompressorKind,
+}
+
+func (kind CompressorKind) String() string {
+	return string(kind)
+}
+
+func (kind *CompressorKind) Set(value string) error {
```

Comment:
Is this function used?

Reply:
this is required as part of the CLI generic flag: https://github.com/urfave/cli/blob/b158c1095ecce00068fafa6d636752bbaeb5f51d/flag_generic.go#L10